src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp

```diff
@@ -440,7 +447,27 @@ PerformanceImplicitGemmForwardV4R4Xdlops::CalculateGemmBBlockCopyPerformancePara
         // calculate threadwise copy size
         auto data_per_thread_copy =
             std::max(1, (GemmKPerBlock * GemmNPerBlock * GemmKPack) / block_size);
-
+        if(miopen::IsEnabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ADD_VECTOR_LOAD_GEMMN_TUNE_PARAM{}))
+        {
+            if(ctx.IsFp16())
```

Comment:
There is divergence between fp16 and other data-type?

Reply:
yes, I tested on fp32 bfp16, there's no big improvement after adding this tunning parameter, the reason, i believe is fp16's the biggest flops, it make LDS operation relatively bottleneck.
so to avoid extra tunning time, i enable this parameter only for fp16.